plugin/src/main/resources/org/jenkinsci/plugins/workflow/cps/Snippetizer/handle-prototype.js

```diff
@@ -1,4 +1,4 @@
-function handlePrototype(url, crumb) {
+function handlePrototype(url, crumb_header, crumb_value) {
```

Comment:
if we have access to the document we should be able to load the crumbs from `document.head`?

Additionally (and I am completely not sure of this) but I think core should handle this for us - we just need to call [`crumb.wrap`](https://www.jenkins.io/blog/2023/05/12/removing-prototype-from-jenkins/#ajax-requests)

e.g. https://github.com/search?q=org%3Ajenkinsci+crumb.wrap&type=code

Reply:
crumb.wrap works in this context. I will clean the rest and update the commit.